.env

```
@@ -6,10 +6,27 @@ API_URL=https://api.vilnacrm.com
 API_PREFIX=/api
 ###< symfony/framework-bundle ###
 
-###> doctrine/doctrine-bundle ###
-DATABASE_URL="mysql://root:root@database:3306/db?serverVersion=11.4"
+###> doctrine/mongodb-odm-bundle ###
+MONGODB_USER=root
+MONGODB_PASSWORD=secret
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGODB_PASSWORD key should go before the MONGODB_USER key

(UnorderedKey)